src/fivetran-destination/src/destination/dml.rs

```diff
@@ -388,6 +407,152 @@ async fn update_file(
     Ok(())
 }
 
+async fn delete_files(
+    schema: &str,
+    table: &Table,
+    file_config: &FileConfig,
+    client: &tokio_postgres::Client,
+    delete_files: &[String],
+) -> Result<(), anyhow::Error> {
+    // TODO(parkmycar): Make sure table exists.
+    // TODO(parkmycar): Retry transient errors.
+
+    // Copy into a temporary table, which we then merge into the destination.
+    let ts = SystemTime::now()
```

Comment:
We have successfully avoided ever using SystemTime except in the `mz_ore::now` package. You need to plumb down a `NowFn` here if we keep this.

Edit: ok, so this is used to generate some tablename. Does that table name need to have guarantees? Like must it always have a higher alphabetical sort order than all tables previously created? There's no guarantee that the clock has progressed in milliseconds since the last time this function was called, so it could presumably generate the same name twice. Is that ok? Can those two tables co-exist?

Could/should this instead be an `AtomicU64` that could have ordering and uniqueness guarantees? What about a uuid? I think there are better choices than a timestamp in general.

Reply:
Probably a better idea also: run the CREATE in a loop with whatever random token you choose so that if a table by that exact name already exists, this will still succeed eventually.

Reply:
The Fivetran Destination doesn't get run in our own infra, it's a binary we send to Fivetran and they run it in their own Docker container. AFAIU they run it in a lamba-like manner where they spin it up every time a sync needs to occur.

You're totally right though, using a timestamp here is a bad approach, I changed this to use the last 12 characters of a UUID like we've done previously, how does that seem to you? In terms of retries, I have a local branch stacked on top of this one which adds error handling at a higher layer, automatically retrying when necessary. Do you mind if I defer the retry handling to that?